examples/pytorch/pointcloud/pointnet/train_partseg_with_visualization.py

```diff
@@ -0,0 +1,232 @@
+import torch
```

Comment:
Can we merge this with `train_partseg.py`? Ideally in the new file, users can control the visualization with some arguments like `--with-visualize`. This makes it much easier to maintain the computation part, i.e. only update one file if needed.

Reply:
Merged